src/lib/vocab/entity/EntityRepo.ts

```diff
@@ -6,21 +6,31 @@ import {PostgresRepo} from '$lib/db/PostgresRepo';
 import type {Entity} from '$lib/vocab/entity/entity';
 import type {EntityData} from '$lib/vocab/entity/entityData';
 import type {ErrorResponse} from '$lib/util/error';
+import {type RowList} from 'postgres';
```

Comment:
```suggestion
import type {RowList} from 'postgres';
```

See this comment for why: https://github.com/feltcoop/felt-server/pull/300/files#r840182574